jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/java/org/jbpm/casemgmt/impl/CaseServiceImplTest.java

```diff
@@ -146,6 +146,7 @@
         processes.add("cases/UserTaskCaseRequiredCaseFileItem.bpmn2");
         processes.add("cases/UserTaskCaseRestrictedCaseFileItem.bpmn2");
         processes.add("cases/UserTaskCaseRequiredRestrictedCaseFileItem.bpmn2");
+        processes.add("cases/UserTaskCaseReadOnlyCaseFileItem.bpmn2");
```

Comment:
It doesn't match with the file name (case sensitive)
```suggestion
        processes.add("cases/UserTaskCaseReadonlyCaseFileItem.bpmn2");
```
or change the file name.

Reply:
file name changed